src/ioctls/vm.rs

```diff
@@ -1584,6 +1612,14 @@ mod tests {
         assert!(vm.set_tss_address(0xfffb_d000).is_ok());
     }
 
+    #[test]
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    fn test_set_identity_map_address() {
+        let kvm = Kvm::new().unwrap();
+        let vm = kvm.create_vm().unwrap();
+        assert!(vm.set_identity_map_address(0xfffb_c000).is_ok());
```

Comment:
Is it possible to test an error case as well, for example this thing from the documentation: `Fails if any VCPU has already been created.`?

Reply:
Done!